backend/services/API.js

```diff
@@ -8,13 +8,21 @@ const reqTimeout = 1000 * 5;
  * specific amount of data according to searchTerm.
  */
 async function getSearchedImages (apiType, limit, searchTerm, offset) {
-  const { link, key, limitString, offsetString } =
-    apiType === apiSources.giphy.id ? apiSources.giphy : apiSources.pixabay;
 
-  const searchQuery = `&q=${searchTerm}`;
-  const query = `&${limitString}=${limit}&${offsetString}=${offset}`;
-  const url = `${link}${key}${searchQuery}${query}`;
+  let url;
+  if (apiType !== "unsplash") {
```

Comment:
please use the constans value for `unsplash` string. hardcoded strings are harder to edit/debug for future fellows.